src/blocks/media-block/ImageBlock.test.tsx

```
@@ -0,0 +1,63 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import * as React from 'react';
+import ImageBlock from './image-block/ImageBlock';
+import '@testing-library/jest-dom/vitest';
+
+// Test utilities are available globally via Vitest config
+
+// Simple mock for framer-motion
+vi.mock('framer-motion', () => ({
+  motion: {
+    div: ({ children, ...props }: any) => <div {...props}>{children}</div>,
+    img: ({ children, ...props }: any) => <img {...props}>{children}</img>,
```

Biome (1.9.4)

[error] img is a void element tag and must not have children.

Unsafe fix: Remove the children.


(lint/correctness/noVoidElementsWithChildren)